libraries/AP_Motors/AP_MotorsHeli.h

```diff
@@ -212,6 +238,9 @@ class AP_MotorsHeli : public AP_Motors {
     AP_Int16        _collective_mid;            // Swash servo position corresponding to zero collective pitch (or zero lift for Asymmetrical blades)
     AP_Int8         _servo_mode;                // Pass radio inputs directly to servos during set-up through mission planner
     AP_Int8         _servo_test;                // sets number of cycles to test servo movement on bootup
+    AP_Float        _collective_hover;          // estimated collective required to hover throttle in the range 0 ~ 1
+    AP_Int8         _collective_hover_learn;    // enable/disabled hover collective learning
+    AP_Int8         _new_integrator;     // enable/disabled new integrator management scheme
```

Comment:
As discussed on the call, let's not use the term, "new".  So either "leaky" or come up with some other name.